WordPress/UITestsFoundation/Screens/Login/LoginEpilogueScreen.swift

```diff
@@ -13,7 +13,7 @@ public class LoginEpilogueScreen: ScreenObject {
         try super.init(
             expectedElementGetters: [loginEpilogueTableGetter],
             app: app,
-            waitTimeout: 60
+            waitTimeout: 70
```

Comment:
Pretty sure there's a better solution other than this one, but I don't know what that is at the moment. I did try playing around with using the loading indicator a while ago but didn't get positive results from that...